script-test/playercomponenttest.js

```diff
@@ -950,37 +951,32 @@ require(
           expect(mockStateUpdateCallback.calls.mostRecent().args[0].data.state).toEqual(MediaState.FATAL_ERROR);
         });
 
-        it('should fire a fatal error on the plugins if the playback type is live', function () {
```

Comment:
Feel like these 2 deleted tests should still exist for:
 - Non-dash
 - Fatal error when not remaining CDN's to failover to

Reply:
Yep good call. We've added an explicit test for not failing over when playing back HLS content.

For the no remaining CDN's, rather than a test getting us failing over multiple times, we've added extra assertions that the CDN url list is being sliced, and there is already a test that we throw fatal error if we only have one CDN, so this should be covered.